buildSrc/src/main/java/com/uber/okbuck/core/manager/RobolectricManager.java

```diff
@@ -115,9 +118,10 @@ public void finalizeDependencies(OkBuckExtension okBuckExtension) {
     API_25("7.1.0_r7", "r1"),
     API_26("8.0.0_r4", "r1"),
     API_27("8.1.0", "4611349"),
-    API_P("P", "4651975"),
```

Comment:
https://github.com/robolectric/robolectric/blob/master/robolectric/src/main/java/org/robolectric/plugins/DefaultSdkProvider.java No longer referenced in robolectric core.